generator/resources/ec2_linux_test_matrix.json

```diff
@@ -21,6 +21,17 @@
     "binaryName": "amazon-cloudwatch-agent.deb",
     "family": "linux"
   },
+  {
+    "os": "ubuntu-23",
+    "username": "ubuntu",
+    "instanceType":"t3a.medium",
+    "installAgentCommand": "go run ./install/install_agent.go deb",
+    "ami": "cloudwatch-agent-integration-test-ubuntu-23*",
```

Comment:
I agree with Seth, it will be nice to know what arch this ami is running.

Reply:
x86_64 will run on both intel and amd hardware. It is confusing that amd the chipmaker is the same name as the arch amd. I will approve if you disagree but I would like you to remove the a in the t3a.medium instance type so we can test on an intel chipset.

Reply:
arch amd = x86_64. Why the confusing naming convention, I do not know.